closed/make/copy/Copy-java.base.gmk

```diff
@@ -35,7 +35,11 @@ TARGETS += \
 	$(INCLUDE_TARGET_DIR)/ibmjvmti.h \
 	#
 
-$(INCLUDE_TARGET_DIR)/jvmti.h : $(TOPDIR)/openj9/runtime/include/jvmti.h
+#$(OUTPUTDIR)/vm/include/jvmti.h : $(OPENJ9_TOPDIR)/runtime/include/jvmti.h.m4 
+#	m4 -D JAVA_SPEC_VERSION=$(VERSION_FEATURE) $< > $@
+
+$(INCLUDE_TARGET_DIR)/jvmti.h : $(TOPDIR)/openj9/runtime/include/jvmti.h.m4
+	m4 -D JAVA_SPEC_VERSION=$(VERSION_FEATURE) $< > $(TOPDIR)/openj9/runtime/include/jvmti.h
```

Comment:
I don't expect this will work: currently, openj9 source is copied into `$(OUTPUTDIR)/vm` and so writing into `$(TOPDIR)/openj9` will only help the _next_ build cycle. I think the rule (commented out) above is correct.

Reply:
I tried to build with the rule above before and let the build stop when building this target, it seems it will copy openj9 source later. When the above rule is completed, only `sourcetool` directory is copied into `$(OUTPUTDIR)`. The later rule will compile in my machine no matter build by Uma  or Cmake.

Reply:
The rule as written doesn't produce the target file and we don't want files written into the source tree.

The dependencies
`j9vm-build -> java.base-copy -> jvmti.h`
will ensure things are done in the right order (even though work is done in parallel).